games/c.yaml

```
@@ -1474,6 +1474,25 @@
   video:
     youtube: rSN1p247J74
 
+- name: Cosmic Kidnap
+  originals:
+  - Cosmic Kidnap
+  type: clone
+  repo: 'https://www.ibiblio.org/pub/micro/pc-stuff/Linux/games/arcade/#:~:text=kidnap'
+  url: 'https://www.ibiblio.org/pub/micro/pc-stuff/Linux/games/arcade/kidnap-0.01.lsm'
+  development: halted
+  status: semi-playable
+  content: open
+  langs:
+  - C
+  frameworks:
+  - SVGALib
+  licenses:
+  - As-is
+  - GPL1
```

```suggestion
  - Custom
```
I couldn't find any license file in the source archive, but I found this in the readme which constitutes a custom license

```
You can modify this game and distribute modified versions providing
that you give me credit. If you make any interesting modifications to
this game, or port it to another platform (I would be quite interested
in DOS or OS/2 ports), please e-mail me and tell me about it and where
to get it (don't actually send me a copy straight away though).
```